packages/tableView/stories/TableView.stories.tsx

```diff
@@ -6,7 +6,7 @@ import DemoTable from "./helpers/DemoTable";
 
 const readme = require("../README.md");
 
-storiesOf("TableView", module)
+storiesOf("Page structure|TableView", module)
```

Comment:
I was undecided about how to categorize this component. I could see it living in any of these three categories:
- Page structure
- Layout
- Data listing